SyncDemo/app/build.gradle

```diff
@@ -0,0 +1,49 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 21
+    buildToolsVersion "21.1.2"
+
+    defaultConfig {
+        applicationId "org.jboss.aerogear.android.cookbook.syncdemo"
+        minSdkVersion 16
+        targetSdkVersion 21
+        versionCode 1
+        versionName "1.0"
+        multiDexEnabled true
+    }
+
+    lintOptions {
+        abortOnError false
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+    }
+    
+    packagingOptions {
+        exclude 'META-INF/INDEX.LIST'
+        exclude 'META-INF/LICENSE'
+        exclude 'META-INF/NOTICE'
+        exclude 'META-INF/ASL-2.0.txt'
+        exclude 'META-INF/LGPL-3.0.txt'
+        exclude 'META-INF/io.netty.versions.properties'        
+    }
+    
+}
+
+dependencies {
+    compile 'com.android.support:appcompat-v7:21.0.3'
+    compile 'com.android.support:multidex:1.0.0'
+    compile 'com.melnykov:floatingactionbutton:1.1.0'    
+    compile 'org.jboss.aerogear:aerogear-android-core:2.0.0'
+    compile 'org.jboss.aerogear:aerogear-android-pipe:2.0.0'
+    compile 'org.jboss.aerogear:aerogear-android-store:2.0.0'
+    compile 'org.jboss.aerogear:aerogear-android-authz:2.0.0'
+    compile (group:'org.jboss.aerogear', name:'sync-client-android', version:'0.0.1-SNAPSHOT', changing:'true' , ext:'aar')
```

Comment:
but isn't that a different artifact ID ?

https://github.com/aerogear/aerogear-android-sync/blob/master/aerogear-android-sync/pom.xml#L23

On Thu, Mar 5, 2015 at 12:35 PM, Daniel Passos <notifications@github.com>
wrote:

> In SyncDemo/app/build.gradle
> <https://github.com/aerogear/aerogear-android-cookbook/pull/49#discussion_r25857149>
> :
>
> > +        exclude 'META-INF/ASL-2.0.txt'
> > +        exclude 'META-INF/LGPL-3.0.txt'
> > +        exclude 'META-INF/io.netty.versions.properties'
> > +    }
> > +
> > +}
> > +
> > +dependencies {
> > +    compile 'com.android.support:appcompat-v7:21.0.3'
> > +    compile 'com.android.support:multidex:1.0.0'
> > +    compile 'com.melnykov:floatingactionbutton:1.1.0'
> > +    compile 'org.jboss.aerogear:aerogear-android-core:2.0.0'
> > +    compile 'org.jboss.aerogear:aerogear-android-pipe:2.0.0'
> > +    compile 'org.jboss.aerogear:aerogear-android-store:2.0.0'
> > +    compile 'org.jboss.aerogear:aerogear-android-authz:2.0.0'
> > +    compile (group:'org.jboss.aerogear', name:'sync-client-android', version:'0.0.1-SNAPSHOT', changing:'true' , ext:'aar')
>
> @matzew <https://github.com/matzew>
> https://github.com/aerogear/aerogear-android-sync/tree/master/aerogear-android-sync
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-android-cookbook/pull/49/files#r25857149>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf